x-pack/plugins/cases/server/services/cases/index.ts

```diff
@@ -330,37 +328,28 @@ export class CasesService {
 
   public async getCases({
     caseIds,
-  }: GetCasesArgs): Promise<SavedObjectsBulkResponse<CaseTransformedAttributes>> {
+  }: GetCasesArgs): Promise<SavedObjectsBulkResponseWithErrors<CaseTransformedAttributes>> {
```

Comment:
Why is this here in the context of this PR?

Reply:
Because the `SavedObjectsBulkResponse` type did not handle the errors returned by the SO client and I wanted the cases client that uses the `getCases` method to use the correct types so it can understand that the response contains errors.